docs/format/index.md

```diff
@@ -175,7 +175,7 @@ Postings sections store monotonically increasing lists of series references that
 └─────────────────────────────────────────┘
 ```
 
-The sequence of postings sections is finalized by an [offset table](#offset-table) pointing to the beginning of each postings section for a given set of label names.
+The sequence of postings sections is finalized by an [offset table](#posting-offset-entry) pointing to the beginning of each postings section for a given set of label names.
 
 ### Offset Table
 
```

Comment:
line below: Let's add that entries of this offset tables can be different depending on series/posting (introduce to further sections `### Posting Offset Entry` and `### Label Offset Entry`